examples/polyfills.js

```diff
@@ -0,0 +1,5529 @@
+ * - _ESAbstract.CreateMethodProperty, License: CC0
+ * - Object.keys, License: MIT
+ * - _ESAbstract.Get, License: CC0
+ * - _ESAbstract.HasOwnProperty, License: CC0
+ * - _ESAbstract.IsCallable, License: CC0
+ * - _ESAbstract.RequireObjectCoercible, License: CC0
+ * - _ESAbstract.SameValueNonNumber, License: CC0
+ * - _ESAbstract.ToBoolean, License: CC0
+ * - _ESAbstract.ToObject, License: CC0
+ * - _ESAbstract.GetV, License: CC0
+ * - _ESAbstract.GetMethod, License: CC0
+ * - _ESAbstract.Type, License: CC0
+ * - Number.isNaN, License: MIT
+ * - Object.isExtensible, License: CC0
+ * - _ESAbstract.CreateIterResultObject, License: CC0
+ * - _ESAbstract.GetPrototypeFromConstructor, License: CC0
+ * - _ESAbstract.OrdinaryCreateFromConstructor, License: CC0
+ * - _ESAbstract.IsConstructor, License: CC0
+ * - _ESAbstract.Construct, License: CC0
+ * - _ESAbstract.IsRegExp, License: CC0
+ * - _ESAbstract.IteratorClose, License: CC0
+ * - _ESAbstract.IteratorComplete, License: CC0
+ * - _ESAbstract.IteratorNext, License: CC0
+ * - _ESAbstract.IteratorStep, License: CC0
+ * - _ESAbstract.IteratorValue, License: CC0
+ * - _ESAbstract.OrdinaryToPrimitive, License: CC0
+ * - _ESAbstract.SameValueZero, License: CC0
+ * - _ESAbstract.ToInteger, License: CC0
+ * - _ESAbstract.ToLength, License: CC0
+ * - _ESAbstract.ToPrimitive, License: CC0
+ * - _ESAbstract.ToString, License: CC0
+ * - Array.of, License: CC0
+ * - Array.prototype.fill, License: CC0
+ * - Array.prototype.find, License: CC0
+ * - Array.prototype.findIndex, License: CC0
+ * - Array.prototype.includes, License: MIT
+ * - Event, License: CC0
+ * - CustomEvent, License: CC0
+ * - Object.getOwnPropertyNames, License: CC0
+ * - String.prototype.endsWith, License: CC0
+ * - String.prototype.includes, License: CC0
+ * - String.prototype.startsWith, License: CC0
+ * - _ESAbstract.ToPropertyKey, License: CC0
+ * - Object.getOwnPropertyDescriptor, License: CC0
+ * - Object.assign, License: CC0
+ * - Symbol, License: MIT
+ * - Symbol.iterator, License: MIT
+ * - _ESAbstract.GetIterator, License: CC0
+ * - Symbol.species, License: MIT
+ * - Map, License: CC0
+ * - Set, License: CC0
+ * - Array.from, License: CC0
+ * - URL, License: CC0-1.0
+ * - Symbol.toStringTag, License: MIT
+ * - Promise, License: MIT
+ * - _ESAbstract.EnumerableOwnProperties, License: CC0
+ * - Object.entries, License: CC0
+ * - _ESAbstract.TrimString, License: CC0
+ * - String.prototype.trim, License: CC0
+ * - _mutation, License: CC0
+ * - DocumentFragment.prototype.append, License: CC0
+ * - DocumentFragment.prototype.prepend, License: CC0
+ * - Element.prototype.after, License: CC0
+ * - Element.prototype.append, License: CC0
+ * - Element.prototype.before, License: CC0
+ * - Element.prototype.prepend, License: CC0
+ * - Element.prototype.remove, License: CC0
+ * - Element.prototype.replaceWith, License: CC0
+*/
+
+(function(self, undefined) {
+  if (typeof Object.entries !== 'undefined') {
+    return;
+  }
```

Comment:
This simple early return prevents executing the polyfills on supported browsers (I assume all browsers we support but IE11 handle `Object.entries`).